actionpack/test/dispatch/routing_test.rb

```diff
@@ -4592,3 +4592,44 @@ def test_positional_args_with_format_false
     assert_equal '/en/posts/2014/12/13', archived_posts_path(2014, 12, 13)
   end
 end
+
+class TestErrorsInController < ActionDispatch::IntegrationTest
```

Comment:
Is there a reason this needs to be a separate test class, unlike every other test in this file?